common/src/main/java/org/opensearch/ml/common/input/MLInput.java

```diff
@@ -140,6 +168,12 @@ public static MLInput parse(XContentParser parser, String inputAlgoName) throws
         List<String> sourceIndices = new ArrayList<>();
         DataFrame dataFrame = null;
 
+        boolean returnBytes = false;
+        boolean returnNumber = true;
```

Comment:
why this is initialized as `true`?

Reply:
By default we should only return number for embeddings. If set it as `false` , no embedding vectors will return.